portal-web/test/functional/com/liferay/portalweb/macros/Publications.macro

```diff
@@ -273,6 +273,14 @@ definition {
 		return ${publicationName};
 	}
 
+	macro hideApplications {
```

Comment:
We don't need to add the macro and path, we can use the existing macro `SystemSettings.editTextAreaSetting`, can you update it?